client/performance-profiler/components/dashboard-content/index.tsx

```diff
@@ -16,6 +17,7 @@ export const PerformanceProfilerDashboardContent = (
 		<div className="performance-profiler-content">
 			<div className="l-block-wrapper">
 				<PerformanceScore value={ 70 } />
+				<CoreWebVitalsDisplay fcp={ 1794 } lcp={ 1966 } cls={ 0.05 } inp={ 391 } ttfb={ 916 } />
```

Comment:
Nice way of calling just passing the values and the component taking from here :)